js/main.js

```diff
@@ -60,59 +64,82 @@
     }
   }
 
+  const animationTranslate = (target, time, endX, endY) => {
+    const startX = -50
+    const startY = -50
+    const diffX = endX - startX
+    const diffY = endY - startY
+    const today = new Date()
+    const clickMillisecond = today.getTime()
+
+    const update = () => {
+      const updateMillisecond = Date.now()
+      const diff = ((updateMillisecond - clickMillisecond))
+      progress = diff / time
+      console.log(diff)
+      progress = Math.min(progress, 1)
+      if (progress >= 0) {
```

Comment:
これ、`progress > 1`のときに変な値を元に計算しないようにしてるんだと思うんだけど、
```suggestion
if (progress < 0 || progress > 1) {
  return;
}
```
これを入れれば`progress`の宣言は`const`にしていいし、`Math.min`も挟まなくてよくなる。